src/rabbitmq.js

```diff
@@ -132,7 +139,8 @@ class RabbitMQ {
       }
       const stringContent = JSON.stringify(content)
       const bufferContent = new Buffer(stringContent)
-      return Promise.resolve(this.channel.sendToQueue(queue, bufferContent))
+      return Promise
+        .resolve(this.publishChannel.sendToQueue(queue, bufferContent))
```

Comment:
it's just that the server acknowledges it, not that the message is passed through and handled by some worker...